testing/test_base.sh

```diff
@@ -50,35 +50,38 @@ echo switch ping $count | tee -a $TEST_RESULTS
 echo %%%%%%%%%%%%%%%%%%%%%% Mud profile tests | tee -a $TEST_RESULTS
 cp misc/system_muddy.conf local/system.conf
 
-device_traffic="tcpdump -en -r inst/run-port-01/scans/monitor.pcap port 47808"
-device_bcast="$device_traffic and ether broadcast"
-device_ucast="$device_traffic and ether dst 9a:02:57:1e:8f:02"
-device_xcast="$device_traffic and ether dst 9a:02:57:1e:8f:03"
-cntrlr_traffic="tcpdump -en -r inst/run-port-02/scans/monitor.pcap port 47808"
-cntrlr_bcast="$cntrlr_traffic and ether broadcast"
-cntrlr_ucast="$cntrlr_traffic and ether dst 9a:02:57:1e:8f:01"
-cntrlr_xcast="$cntrlr_traffic and ether dst 9a:02:57:1e:8f:03"
+port_1_traffic="tcpdump -en -r inst/run-port-01/scans/monitor.pcap port 47808"
+port_1_bcast_2="$port_1_traffic and ether src 9a:02:57:1e:8f:02 and ether broadcast"
+port_1_ucast_2="$port_1_traffic and ether src 9a:02:57:1e:8f:02 and ether dst 9a:02:57:1e:8f:01"
+port_1_bcast_3="$port_1_traffic and ether src 9a:02:57:1e:8f:03 and ether broadcast"
+port_1_ucast_3="$port_1_traffic and ether src 9a:02:57:1e:8f:03 and ether dst 9a:02:57:1e:8f:01"
+port_2__traffic="tcpdump -en -r inst/run-port-02/scans/monitor.pcap port 47808"
+port_2_bcast_1="$port_2_traffic and ether src 9a:02:57:1e:8f:01 and ether broadcast"
+port_2_ucast_1="$port_2_traffic and ether src 9a:02:57:1e:8f:01 and ether dst 9a:02:57:1e:8f:02"
+port_2_bcast_3="$port_2_traffic and ether src 9a:02:57:1e:8f:03 and ether broadcast"
+port_2_ucast_3="$port_2_traffic and ether src 9a:02:57:1e:8f:03 and ether dst 9a:02:57:1e:8f:02"
 
 function test_mud {
     type=$1
     echo %%%%%%%%%%%%%%%%% test mud profile $type
     cmd/run -s device_specs=misc/device_specs/bacnet_$type.json
     echo result $type $(sort inst/result.log) | tee -a $TEST_RESULTS
-    bcast=$($device_bcast | wc -l)
-    ucast=$($device_ucast | wc -l)
-    xcast=$($device_xcast | wc -l)
-    echo device $type $(($bcast > 2)) $(($ucast > 2)) $(($xcast > 0)) | tee -a $TEST_RESULTS
-    bcast=$($cntrlr_bcast | wc -l)
-    ucast=$($cntrlr_ucast | wc -l)
-    xcast=$($cntrlr_xcast | wc -l)
-    echo cntrlr $type $(($bcast > 2)) $(($ucast > 2)) $(($xcast > 0)) | tee -a $TEST_RESULTS
+    bcast_2=$($port_1_bcast_2 | wc -l)
+    ucast_2=$($port_1_ucast_2 | wc -l)
+    bcast_3=$($port_1_bcast_3 | wc -l)
+    ucast_3=$($port_1_ucast_3 | wc -l)
+    echo port_1 $type $(($bcast_2 > 2)) $(($ucast_2 > 0)) $(($bcast_3 > 2)) $(($ucast_3 > 0)) | tee -a $TEST_RESULTS
```

Comment:
I'm wondering if these all should be >0 -- not sure why they were >2 initially. What's the intent here?

Reply:
I just copied it from the origin test... I thought it was something related to the flooding for learned destination.. But this explanation does not seem to make sense... Changed to 0.